src/Akka.Persistence.MongoDb/Query/MongoDbReadJournal.cs

```diff
@@ -24,23 +28,21 @@ public class MongoDbReadJournal : IReadJournal,
         private readonly TimeSpan _refreshInterval;
         private readonly string _writeJournalPluginId;
         private readonly int _maxBufferSize;
-        
+        private readonly ExtendedActorSystem _system;
+
+        private readonly object _lock = new object();
+        private IPublisher<string> _persistenceIdsPublisher;
+
         /// <inheritdoc />
         public MongoDbReadJournal(ExtendedActorSystem system, Config config)
         {
             _refreshInterval = config.GetTimeSpan("refresh-interval");
             _writeJournalPluginId = config.GetString("write-plugin");
-            _maxBufferSize = config.GetInt("max-buffer-size");
-        }
+            _maxBufferSize = config.GetInt("max-buffer-size"); 
+            _system = system;
 
-        /// <summary>
-        /// Returns a default query configuration for akka persistence SQLite-based journals and snapshot stores.
-        /// </summary>
-        /// <returns></returns>
-        public static Config DefaultConfiguration()
-        {
-            return ConfigurationFactory.FromResource<MongoDbReadJournal>(
-                "Akka.Persistence.MongoDb.reference.conf");
+            _lock = new ReaderWriterLockSlim();
```

Comment:
Just use a `lock` keyword - no need for a `ReaderWriterLock`

Reply:
Just delete this line, `_lock` is already initialized in the code above